go.mod

```diff
@@ -0,0 +1,78 @@
+module github.com/openebs/cstor-csi
+
+go 1.13
+
+require (
+	github.com/container-storage-interface/spec v1.2.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/protobuf v1.3.2
+	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/kubernetes-csi/csi-lib-iscsi v0.0.0-20200118015005-959f12c91ca8
+	github.com/kubernetes-csi/csi-lib-utils v0.7.0
+	github.com/openebs/api v0.0.0-20200304155919-9ac1d4814a3f
+	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5
+	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	golang.org/x/sys v0.0.0-20200122134326-e047566fdf82
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	golang.org/x/tools v0.0.0-20191029190741-b9c20aec41a5 // indirect
+	google.golang.org/grpc v1.26.0
+	k8s.io/api v0.17.3
+	k8s.io/apimachinery v0.17.3
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/code-generator v0.17.3
+	k8s.io/kubernetes v1.17.3
+	k8s.io/utils v0.0.0-20200124190032-861946025e34
+)
+
+replace (
+	github.com/openebs/api => /home/prateek/gocode/src/github.com/openebs/api
```

Comment:
nit: is there any dependency on any PR for openebs/api repo?

Reply:
that intentional, will remove once the api changes merged in openebs/api repo and remove while pushing the vendor update.